mnefun/_mnefun.py

```diff
@@ -195,11 +222,22 @@ class Params(Frozen):
     See also
     --------
     do_processing
+    run_sss_localy
+    mne.maxwell_filter
 
     Notes
     -----
-    Params has additional properties. Use ``dir(params)`` to see
+    - Params has additional properties. Use ``dir(params)`` to see
     all the possible options.
+    - For Maxwell filtering with mne.maxwell_filter the following default
+    parameters:
+        * tSSS correlation = 0.98
+        * Order of internal component of spherical expansion = 8
+        * Order of external component of spherical expansion = 3
+        * tSSS buffer length = 60 seconds
+        * Spherical expansion coordinate frame = head
+        * Coordinate frame origin =  head-digitization-based origin fit
```

Comment:
For example, we were already using a head-digitization-based origin fit when using MaxFilter. In principle we should be able to do the same things with both (except that eventually mne-python will be more capable, hopefully).